src/util/psbt/map/input.rs

```diff
@@ -67,7 +67,7 @@ pub struct Input {
     pub witness_script: Option<Script>,
     /// A map from public keys needed to sign this input to their corresponding
     /// master key fingerprints and derivation paths.
-    pub hd_keypaths: BTreeMap<PublicKey, KeySource>,
+    pub bip32_derivation: BTreeMap<PublicKey, KeySource>,
```

Comment:
Hmm, is this a necessary breaking change? I recognize that there's a bunch of PSBT related breaking changes being combined, just wondering if this one is significant enough.

I do like the mention of bip32 in there, though, so ack.

Reply:
This is a part of the other PR which is needed to be accepted first https://github.com/rust-bitcoin/rust-bitcoin/pull/513

The rationale is to get all names matching BIP spec